packages/library-base/src/commonMain/kotlin/io/realm/dynamic/DynamicMutableRealmObject.kt

```diff
@@ -45,4 +46,20 @@ public interface DynamicMutableRealmObject : DynamicRealmObject {
      * name, or if the value doesn't match the [RealmStorageType.kClass] type of the property.
      */
     public fun <T> set(propertyName: String, value: T): DynamicMutableRealmObject
+    public fun set(vararg pairs: Pair<String, Any?>): DynamicMutableRealmObject {
```

Comment:
Just like for `DynamicMutableRealmObject.create` it is just convenience for setting multiple properties in one go
```
object.set(
    "id" to 1,
    "name" to "NAME,
)
```
instead of 
```
object.apply {
    set("id", 1)
    set("name", "NAME)
}
```